### PR TITLE
Rust-Wechaty

### PR DESCRIPTION
**Pull request by shraddhavp**
this pr links to  the updated version of Rust-Wechaty

### CODE REVIEW

File: docusaurus/docs/polyglot/rust/overview.md [hunk 1]
```diff
@@ -22,12 +21,12 @@ make bot
 
 ## Blogs
 
-Read blogs with the `rust` tag at <https://wechaty.js.org/tags.html#rust>
+Read blogs with the `rust` tag at [here](https://wechaty.js.org/tags.html#rust).
```
Comment (Soumi7) [inline docusaurus/docs/polyglot/rust/overview.md]: remove "at"

**Review by Soumi7 — APPROVED** [review]
LGTM

File: docusaurus/docs/polyglot/rust/overview.md (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/polyglot/rust/overview.md]: done

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM

Please follow #1257 and feel free to help yourself to merge this great fix, thank you very much!

**Comment by shraddhavp** [comment]
> LGTM
> 
> Please follow #1257 and feel free to help yourself to merge this great fix, thank you very much!

thanks, Huan @huan

**Comment by huan** [comment]
You are welcome!

**Pull request merged**